Backpack-SwiftUI/Nudger/Classes/BPKNudger.swift

```diff
@@ -163,12 +174,27 @@ public struct BPKNudger: View {
         value = max(value - step, minValue)
         updateButtonStates()
     }
+    
+    private func accessibilityIdentifier(for label: String) -> String {
+        if let prefix = accessibilityPrefix {
+            return "\(prefix)_\(label)"
```

Comment:
is the underscore needed? or should a space be better? also, how does this work for rtl languages? I'm not sure this is the right way to go about this. @gert-janvercauteren you thik that's a problem for voice over? or is VO smart enough?

Reply:
that's a identifier, not a label, so, there won't be a case with the issue here :)